brage-import/src/main/java/no/sikt/nva/brage/migration/merger/CristinImportPublicationMerger.java

```diff
@@ -216,10 +216,14 @@ private Set<AdditionalIdentifier> mergeAdditionalIdentifiers() {
     }
 
     private AssociatedArtifactList determineAssociatedArtifacts() {
-        if (existingPublication.getAssociatedArtifacts().isEmpty()) {
+        if (existingAssociatedArtifactsAreEmpty()) {
             return bragePublicationRepresentation.publication().getAssociatedArtifacts();
         }
-
+        if (existingPublicationHasNoPublishedFiles()) {
+            var mergedAssociatedArtifacts = new ArrayList<>(existingPublication.getAssociatedArtifacts());
+            mergedAssociatedArtifacts.addAll(bragePublicationRepresentation.publication().getAssociatedArtifacts());
+            return new AssociatedArtifactList(mergedAssociatedArtifacts);
+        }
```

Comment:
I think we want to do this -> if existin publication does not have published files -> merge associatedArtifacts